src/sentry/static/sentry/app/views/settings/project/projectProcessingIssues.jsx

```diff
@@ -147,6 +140,7 @@ const ProjectProcessingIssues = createReactClass({
           error: false,
           loading: expected > 0,
         });
+        // TODO (billyvg): Need to fix this
```

Comment:
did you want to fix in this pr or at some point later? (mostly just making sure you know these are still here)

Reply:
👍 Some point later